Docs/topics/discord/information.md

```diff
@@ -3,43 +3,31 @@ channelId: 1049224956162097212
-5. Please keep criticism constructive. We generally won't take action for this, but it will be noticed if you are consistently negative when discussing topics with other members.
-6. Discussion of piracy won't be tolerated. LittleBigRefresh is not the place to discuss this type of thing. We won't mute you for simply mentioning piracy, but we definitely will take action if you're discussing methods to enable piracy, or suggesting that people commit piracy. Just keep it out of this server.
-7. Try to avoid discussing topics relating to politics, religion and world tragedies. Those topics can quickly cause discomfort and start arguments. Remember, this is a LittleBigPlanet server; we're here to discuss and play LittleBigPlanet!
+3. Treat everyone with respect. Harassment, witch hunting, rage baiting, sexism, racism, hate speech, any other form of hateful conduct or targeting, will not be tolerated.
+4. No mature or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other inappropriate material.
+5. Please keep criticism and conversation civil; don’t be intentionally rude or abrasive. We do our best to be understanding, but will take note of consistently unpleasant behavior.
+6. Endorsement or admission of piracy won't be tolerated. We won't mute you for mention of the subject, but we will take action if you're discussing methods or encouraging others to pirate.
+7. Try to avoid discussing hot topics (drama/politics/religion/recent events, etc.) We’re all here to play and discuss a great game series we enjoy, it's best not to dwell on unrelated topics that may divide or embitter.
 
-Keep in mind that this list is not exhaustive of things we might punish for. That said, just use common sense and you'll be okay.
+This list is non-exhaustive of behavior we might take action on, but in effect, use common sense and all will be well. If you have any questions about our rules feel free to contact a staff member; if you have suggestions you can create a GitHub issue on our [documentation repo.](<https://github.com/LittleBigRefresh/Docs/issues/new>)
 
 ## Enforcement
 
-We generally try to be hands-off when it comes to punishment. In most cases, the offending content will simply be deleted, or you will receive a verbal warning. In more extreme cases, you will be muted for anywhere between an hour and a week. This mostly happens after persisting and being told to stop doing something by a staff member.
+Generally, we try to be hands-off when it comes to punishment. Mistakes happen and everyone has off days–more often than not we might simply ask you to stop doing something and the offending content will be deleted. In more extreme cases you might receive a verbal warning from our bot–again this isn’t a punishment so to speak but something we issue in the hopes that behavior changes.
```

Comment:
> In more extreme cases you might receive a verbal warning from our bot

Our bot doesn't do this; (unless Dyno has been doing that without my knowledge?) all actions apart from DO are manual.